src/MongoDB.Driver.Core/Core/Operations/CountDocumentsOperation.cs

```diff
@@ -27,6 +27,7 @@ namespace MongoDB.Driver.Core.Operations
 {
     /// <summary>
     /// Represents a count documents operation.
+    /// For a fast estimate of the total documents in a collection see <see cref="CountOperation"/>.
```

Comment:
For some reason my IDE did NOT do a compile time check on this cref (previously it said `EstimatedDocumentCountOperation` which does not exist)